### PR TITLE
Add log message HTTP request and response to AG server

### PR DESCRIPTION
**Pull request by pnt325**
Log URL and response code message for HTTP request to AG server

### CODE REVIEW

File: src/AgApiClient.cpp [hunk 1]
```diff
@@ -112,18 +116,22 @@ bool AgApiClient::postToServer(String data) {
   String uri =
       "http://hw.airgradient.com/sensors/airgradient:" + ag->deviceId() +
       "/measures";
-  logInfo("Post uri: " + uri);
-  logInfo("Post data: " + data);
+  // logInfo("Post uri: " + uri);
```
Comment (airgradienthq) [inline src/AgApiClient.cpp:119]: remove?

File: src/AgApiClient.cpp [hunk 2]
```diff
@@ -112,18 +116,22 @@ bool AgApiClient::postToServer(String data) {
...
+  // logInfo("Post data: " + data);
```
Comment (airgradienthq) [inline src/AgApiClient.cpp:120]: remove?

File: src/AgApiClient.cpp [hunk 3]
```diff
@@ -112,18 +116,22 @@ bool AgApiClient::postToServer(String data) {
...
   HTTPClient client;
   if (client.begin(wifiClient, uri.c_str()) == false) {
+    logError("Init client failed");
     return false;
   }
   client.addHeader("content-type", "application/json");
   int retCode = client.POST(data);
   client.end();
 
+  logInfo(String("POST: ") + uri);
```
Comment (pnt325) [inline src/AgApiClient.cpp:132]: > remove?

@airgradienthq 
Add new log here

File: src/AgApiClient.cpp (hunk 2 above)
Comment (pnt325) [reply src/AgApiClient.cpp:120]: I'm replace by new log bellow

**Pull request merged**